### DISCUSSION TITLE
PowerPC

### DISCUSSION THREAD

**Discussion by DjDeets** [ptitSeb/box86#365] [Q&A]
Does anybody know if Box86 can run x86 programs on old apple big endian powerpc architectures? I run void on my 64 bit PowerMac G5 so I'm specifically interested in that. But for sake of thoroughness, what about 32 bit processors such as a G4 and modern day little endian Open Power processors?

**Comment by ptitSeb** [comment]
It will not work. Box86 needs a little-endian CPU to work.
(note that box86 does work on littleendian power9 cpu with the POWER9 switch)

**Reply by FreddyFeuerstein** [reply]
so a g4 could run it, right?

**Reply by ptitSeb** [reply]
Check with the VoidLinux community.
Look here for example: https://twitter.com/octaforge/status/1340079199843995651
